src/views/Vacancies/components/FilterSidebar.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField, Typography, InputAdornment } from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+import CloseIcon from "@material-ui/icons/Close";
+
+import FacetFilter from "./FacetFilter";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    borderRight: "1px solid #e0e0e0",
+    background: "#f1f5f5",
+    padding: theme.spacing(3),
+    height: "100%",
+  },
+  group: {
+    marginBottom: theme.spacing(3),
+  },
+  groupLabel: {
+    textTransform: "uppercase",
+  },
+  closeIcon: {
+    cursor: "pointer",
+  },
+}));
+
+interface CompProps {
+  itemsTotal: number;
+  facetGroups: any;
+  searchFilters: any;
```

Comment:
strong type for facetGroups and searchFilters

Reply:
Strong typing has been added here.

https://github.com/vonq/saleor-storefront/blob/dce4242590710005d34bd31f59a352c647c87266/src/core/apiLayer/vacancyService/types.ts#L1-L35